<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkUSNavigationStepZoneMarking</class>
 <widget class="QWidget" name="QmitkUSNavigationStepZoneMarking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="bodyMarkerTrackingStatusLabel">
     <property name="styleSheet">
      <string notr="true">
              background-color: #ff7878; margin-right: 1em; margin-left: 1em;
              border: 1px solid grey
            </string>
     </property>
     <property name="text">
      <string>Body marker is not inside the tracking volume.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>You can mark critical structures by navigating to them using the ultrasound probe and freezing the image.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QmitkUSNavigationFreezeButton" name="freezeButton"/>
   </item>
   <item>
    <widget class="QLabel" name="zoneAddingExplanationLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Now you can add a critical structure by clicking on its center in the ultrasound image, moving the mouse until the arc radius is as desired and clicking again then.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Critical Structures</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QmitkUSZoneManagementWidget" name="zonesWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkUSZoneManagementWidget</class>
   <extends>QWidget</extends>
   <header>src/internal/Widgets/QmitkUSZoneManagementWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkUSNavigationFreezeButton</class>
   <extends>QPushButton</extends>
   <header>src/internal/Widgets/QmitkUSNavigationFreezeButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/USNavigation.qrc"/>
 </resources>
 <connections/>
</ui>
